src/modules/billing/mappers/api/invoice.js

```diff
@@ -16,10 +16,10 @@ const map = {
   'totals.netTotal': 'netTotal',
   'invoiceLicences[].licence.licenceNumber': 'licenceNumbers[]',
   'financialYear.yearEnding': 'financialYearEnding',
-  invoiceLicences: {
```

Comment:
I think this code was working because it divides the invoices into water companies/other abstractors in the UI:
```
  invoiceLicences: {
    key: 'isWaterUndertaker',
    transform: containsWaterUndertaker
  }
```

Reply:
Apologies, didn't realise I'd changed it. Thank you for pointing it out :)